api/v1/middleware/ratelimiting.js

```diff
@@ -0,0 +1,13 @@
+const rateLimiter = require('express-rate-limit-middleware').rateLimit
+
+const config = require('../../config');
+const redisStorage = require('express-rate-limit-middleware').redisRateLimit
+const redisClient = require('../../cache').instance();
+
+module.exports = (req, res, next) => {
```

Comment:
Looking at the documentation of the module you're using, the `rateLimiter`  object itself is middleware, so you should be able to unwrap this further into:
```
module.exports = rateLimiter({
...
});
```